app/src/main/java/org/wikipedia/page/PageActivity.kt

```diff
@@ -110,23 +110,27 @@ class PageActivity : BaseActivity(), PageFragment.Callback, LinkPreviewDialog.Ca
         clearActionBarTitle()
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
         binding.pageToolbarButtonSearch.setOnClickListener {
+            pageFragment.articleInteractionEvent.logSearchWikipediaClick()
```

Comment:
All of these interactions will crash if they happen before the article is loaded, since `articleInteractionEvent` will not yet be initialized.  Instead of making it `lateinit`, let's make it nullable, and change all these calls to `articleInteractionEvent?.log...`